torchfields/voting.py

```diff
@@ -48,19 +48,40 @@ def vote(self, softmin_temp=1, blur_sigma=1):
         DisplacementField of shape (1, 2, H, W) containing the vector
         voting result
     """
+    field_weights = get_field_weights(self, 
+                                      softmin_temp=softmin_temp,
+                                      blur_sigma=blur_sigma)
+    partition = field_weights.sum(dim=0, keepdim=True)
+    field_weights = field_weights / partition
+    return (self * field_weights.unsqueeze(-3)).sum(dim=0, keepdim=True)
+
+def get_field_weights(field, softmin_temp=1, blur_sigma=1):
```

Comment:
The name `get_field_weights` seems a bit ambiguous.
Should the function name be more specific so that it's clear that it refers to *vector voting* weights?
I'm not sure...